src/element/resizeElements.ts

```diff
@@ -105,13 +104,6 @@ export const transformElements = (
       );
     }
 
-    // update cursor
-    // FIXME it is not very nice to have this here
-    document.documentElement.style.cursor = getCursorForResizingElement({
-      element,
-      transformHandleType,
-    });
-
```

Comment:
From testing it doesn't seem like we need it, but there may be some edge cases. If so, we'll move it up to `App.tsx`.